exporter/prometheusremotewriteexporter/config.go

```diff
@@ -52,6 +52,9 @@ type Config struct {
 
 	// SendMetadata controls whether prometheus metadata will be generated and sent
 	SendMetadata bool `mapstructure:"send_metadata"`
+
+	// SendRW2 controls whether prometheus remote write v1 or v2 is sent
+	SendRW2 bool `mapstructure:"send_remote_write_v2"`
```

Comment:
I wouldn't put a boolean here, but an ENUM. We expect PRW to evolve into 2.1, 2.2, etc. If we add a boolean, which version of PRv2 are we sending?

Reply:
will updated

Reply:
fixed